trinity/chains/mainnet.py

```diff
@@ -1,5 +1,6 @@
-from evm.chains.mainnet import (
+from evm.chains.mainnet import (  # noqa: F401
     BaseMainnetChain,
+    MainnetChain,
```

Comment:
I assume so that you can import it from here?  That feels weird if it's the case.

Reply:
Yeah, it seemed more natural for the nodes to import both the `MainnetLightDispatchChain` and the `MainnetChain` from the same trinity module. But I don't feel strongly about it, so I'll roll it back.

Reply:
I think my preference would be to:

1. Try to keep this practice constrained to `__init__` files.
2. Explicitly comment anytime we do it outside of an `__init__` file (but I'm kind of inclined to just veto this as something we do)